core/chains/evm/txmgr/txmgr.go

```diff
@@ -222,10 +233,10 @@ func (b *Txm[CHAIN_ID, HEAD, ADDR, TX_HASH, BLOCK_HASH, R, SEQ, FEE]) Reset(call
 	return err
 }
 
-// abandon, scoped to the key of this txm:
+// Abandon, scoped to the key of this txm:
```

Comment:
Is this only exported for test access? If so, could use a test file wrapper instead:
```go
// helpers_test.go

func (b *Txm[CHAIN_ID, HEAD, ADDR, TX_HASH, BLOCK_HASH, R, SEQ, FEE]) Abandon(addr ADDR) (err error) {
	return b.abandon(addr)
}
```

Reply:
```suggestion
// abandon, scoped to the key of this txm:
```